blog/articles/event-drive-app-knative-eventing-kogito.md

```diff
@@ -0,0 +1,130 @@
+---
+Title: 'Orchestrating Events with Knative and Kogito'
+Author: Ricardo Zanini and Tihomir Surdilovic
+Author handle: https://twitter.com/zaninirica, https://twitter.com/tsurdilo
+Date: ''
+Description: A brief introduction of Knative Eventing integration with Kogito implementation of the Serverless Workflow specification
+Folder with media files: 'images/kogito'
+Blog URL: ''
+Labels: Demo
+Reviewers: ''
+
+---
+| Reviewer  | Date | Approval |
+| ------------- | ------------- | ------------- |
+| <!-- Your Github handle here -->  |   |:+1:, :monocle_face:, :-1:  |
+| <!-- Your Github handle here --> |   |  |
```

Comment:
Note: Add me when merging :slightly_smiling_face: anyone else who needs to sign off?